### ISSUE TITLE
math.derivative returns a strange form of a polynomial involving negative powers.

### ISSUE THREAD

**Issue by deSelbyPsyche** [josdejong/mathjs#3072]
**Describe the bug**
The bug is that when using math.derivative to evaluate `1x^2 -4x^-2`, mathjs returns: `(2 * x ^ 4 + 8) / x ^ 3`. I appreciate that this is one valid way of notating the answer, however, it is quite strange. An analogy would be that if an arithmetic function evaluated `2+2` and returned `8/2` we would likely consider that a bug. It is true that `8/2` is one way of notating the answer but it's not how the vast majority of users would expect to receive the result, which would be `4`.

It should be: 2 * x + 8 * x ^ -3, or `2 * x + 8 / x ^ 3`. The documentation includes some examples of derivatives, but not expressions like this, so maybe it was overlooked.

As a comparison the Python library SymPy evaluates `1*x**2 -4*x**-2`to: `2*x + 8/x**3`.

**To Reproduce**
Call math.derivative with the above polynomial and see that it evaluates to the above result.

**Comment by josdejong** [comment]
Makes sense. I'll label this as a possible improvement since it is not strictly speaking a bug.

Anyone interested in looking into this?
